dom.bs

```diff
@@ -2206,16 +2208,17 @@ steps:
 
 <h5 id=signaling-slot-change>Signaling slot change</h5>
 
-<p>Each <a>unit of related similar-origin browsing contexts</a> has a
-<dfn export>signal slot list</dfn> (a list of <a>slots</a>). Unless stated otherwise it is empty.
-[[!HTML]]
+<p>Each <a>similar-origin window agent</a> has a <dfn export>signal slot list</dfn> (a list of
+<a>slots</a>). Unless stated otherwise it is empty. [[!HTML]]
 
 <p>To <dfn noexport>signal a slot change</dfn>, for a <a>slot</a> <var>slot</var>, run these steps:
 
 <ol>
- <li><p>If <var>slot</var> is not in <a>unit of related similar-origin browsing contexts</a>'
- <a>signal slot list</a>, append <var>slot</var> to
- <a>unit of related similar-origin browsing contexts</a>' <a>signal slot list</a>.
+ <li><p>Let <var>agent</var> be the <a>similar-origin window agent</a> to which <var>slot</var>'s
+ <a>relevant Realm</a> belongs.
```

Comment:
We may want to consider introducing a shorthand in HTML so you can say "slot's relevant agent".